features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -86,6 +86,21 @@ class WisunBorderRouter {
      * \return MESH_ERROR_NONE on success.
      * \return MESH_ERROR_UNKNOWN in case of failure.
      * */
+    mesh_error_t start(WisunInterface *mesh_if, NetworkInterface *backbone_if);
+
+    /**
+     * \brief Start Wi-SUN Border Router
+     *
+     * Starts Wi-SUN Border Router and routing between the mesh and backbone interfaces. Network interfaces
+     * must be initialized and connected before calling the start. Backbone interface can be either Ethernet
+     * (EMAC) or Cellular.
+     *
+     * \param mesh_if Wi-SUN mesh network interface
+     * \param backbone_if Backbone network interface
+     * \return MESH_ERROR_NONE on success.
+     * \return MESH_ERROR_UNKNOWN in case of failure.
+     * */
+    MBED_DEPRECATED_SINCE("mbed-os-5.15", "Using NetworkInterface type for mesh_if is deprecated, use WisunInterface instead")
```

Comment:
Please update the deprecated version to match the next Mbed OS 5.15 version, 5.15.8?.